### DISCUSSION TITLE
docker-related installation question

### DISCUSSION THREAD

**Discussion by sudo-joe** [blakeblackshear/frigate#4200] [General]
Hello

hoped to find a baremetall installer for Frigate, but it seems i have to learn Docker....

Any ideas what I am doing wrong?

Thank's  alot for any feedback!
Joe

mint@mint:~/Desktop/frigate-master$ ls
benchmark.py  docker-compose.yml  frigate       LICENSE   migrations       README.md             requirements.txt         web
docker        docs                labelmap.txt  Makefile  process_clip.py  requirements-dev.txt  requirements-wheels.txt

mint@mint:~/Desktop/frigate-master$ docker-compose up -d
ERROR: The Compose file './docker-compose.yml' is invalid because:
Unsupported config option for services.dev: 'group_add'
mint@mint:~/Desktop/frigate-master$ docker-compose up 
ERROR: The Compose file './docker-compose.yml' is invalid because:
Unsupported config option for services.dev: 'group_add'
mint@mint:~/Desktop/frigate-master$

**Comment by NickM-27** [comment]
You should not be running frigate from the git repo and this is not a supported way to run production frigate.

Just follow the install docs in a fresh directory: https://docs.frigate.video/installation#docker
